qiskit/scheduler/config.py

```diff
@@ -31,5 +30,5 @@ def __init__(self, inst_map: InstructionScheduleMap, meas_map: List[List[int]],
             dt: Sample duration.
         """
         self.inst_map = inst_map
-        self.meas_map = format_meas_map(meas_map)
+        self.meas_map = meas_map
```

Comment:
removed `format_meas_map` .
When I make `target` class like this
```python
Target(meas_map=schedule_config.meas_map)
```

the type of `meas_map` is restricted to `list` (https://github.com/Qiskit/qiskit-terra/pull/10258#discussion_r1233237507).